recipe/meta.yaml

```diff
@@ -7,24 +7,22 @@ package:
   version: {{ version|replace("-", "_") }}
 
 source:
-  fn: reshape_{{ version }}.tar.gz
   url:
     - {{ cran_mirror }}/src/contrib/reshape_{{ version }}.tar.gz
     - {{ cran_mirror }}/src/contrib/Archive/reshape/reshape_{{ version }}.tar.gz
   sha256: 4d5597fde8511e8fe4e4d1fd7adfc7ab37ff41ac68c76a746f7487d7b106d168
 
 build:
-  merge_build_host: true  # [win]
-  number: 2
-  skip: true  # [win32]
+  merge_build_host: True  # [win]
+  number: 3
   rpaths:
     - lib/R/lib/
     - lib/
 
 requirements:
   build:
-    - {{ compiler('c') }}        # [not win]
-    - {{ compiler('m2w64_c') }}        # [win]
```

Comment:
This recipe contains compiled code, but all the builds pass without explicitly specifying the compilers. I assume this means they are being provided by one of the dependencies, presumably r-base

Reply:
Sounds reasonable.